modules/spacelift-automation/README.md

```diff
@@ -0,0 +1,191 @@
+# `spacelift-automation`
```

Comment:
@gberenice we're 100% aligned on both of those needs re: managing w/ IaC + automating updates. We can prioritize them as "next up", but as with all open source work we do (other than maintenance), that doesn't mean it'll happen tomorrow. 

I'm of the mind that we treat this repo as just the spacelift-automation child module and we have other spacelift related child modules in their own repo. Let's build for our future state rather than build for convenience today. 

Sound good to you?

Reply:
Yes, let's do that. Updated.